### PR TITLE
feat: Single district view

### LINKED ISSUE #14: Create view for single districts
This will require:
* A button in the presentation selection menu
* Either a dropdown for choosing an available county based on the dataset, or a search box in the presentation settings menu
* A view that presents a chosen county

### PR DESCRIPTION
**Pull request by simendag**
# Description
A requested feature is to be able to view singular districts and how parties fared there. This is a first draft/attempt at that view. Be thorough in the review!
# Testing
## Setup
Make sure to use the browser console and clear the application data. State has been changed and I am unsure how that will impact your testing.
## Expected
* There is a new button for accessing single district presentation
* When clicked, a table of a single district appears in the presentation section, and relevant settings appear in presentation settings
* Changing the new dropdown select results in changing the view with accurate results
* The button opens the default (Østfold, 2017 as of now)
* State is correctly stored and loaded as in all other views
* Page size is set to the number of parties that got mandates in the election (not just in the district), keeping the view stable

Closes #14

### CODE REVIEW

File: src/app/Layout/PresentationSettings/PresentationReducer.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { Action } from "redux";
 import { PresentationAction as KnownAction } from "./PresentationActions";
 import { PresentationAction } from "../Types/ActionTypes";
 
-export function presentationReducer(state: PresentationState | undefined, incomingAction: Action) {
+export function presentationReducer(state: PresentationState | undefined, incomingAction: Action): PresentationState {
```
Comment (Log234) [inline src/app/Layout/PresentationSettings/PresentationReducer.ts]: These changes were not strictly related to this PR.

File: src/app/Layout/PresentationSettings/PresentationSettings.tsx [hunk 2]
```diff
@@ -5,32 +5,39 @@ import { PresentationType } from "../Types/PresentationType";
...
     decimals: string;
     showPartiesWithoutSeats: boolean;
     changeDecimals: (decimals: string, decimalsNumber: number) => void;
     toggleShowPartiesWithoutSeats: (event: React.ChangeEvent<HTMLInputElement>) => void;
+    selectDistrict: (event: React.ChangeEvent<HTMLSelectElement>) => void;
     results: LagueDhontResult;
 }
 export class PresentationSettings extends React.Component<PresentationSettingsProps> {
     static defaultProps = {} as PresentationSettingsProps;
-    onChange() {
```
Comment (Log234) [inline src/app/Layout/PresentationSettings/PresentationSettings.tsx]: These changes were not strictly related to this PR either.

File: src/app/Layout/PresentationSettings/PresentationSettings.tsx [hunk 3]
```diff
@@ -5,32 +5,39 @@ import { PresentationType } from "../Types/PresentationType";
...
+            this.props.currentPresentation === PresentationType.ElectionTable ||
+            this.props.currentPresentation === PresentationType.SingleCountyTable
         ) {
             return true;
         }
         return false;
     }
 
+    needsDistrictDropdown(): boolean {
+        if (this.props.currentPresentation === PresentationType.SingleCountyTable) {
```
Comment (Log234) [inline src/app/Layout/PresentationSettings/PresentationSettings.tsx]: This line should be split, so that "this.props..." starts at the line below.

**Comment by simendag** [comment]
The slider/decimal issue should be fixed now.

File: src/app/Layout/PresentationSettings/PresentationSettings.tsx (hunk 2 above)
Comment (simendag) [reply src/app/Layout/PresentationSettings/PresentationSettings.tsx]: I should definitely get better at using range reverts of files before staging.

**Pull request merged**